plugins/linux-pipewire/CMakeLists.txt

```diff
@@ -33,14 +38,7 @@ target_sources(
           screencast-portal.c
           screencast-portal.h)
 
-target_link_libraries(linux-pipewire PRIVATE OBS::libobs OBS::obsglad PipeWire::PipeWire GIO::GIO Libdrm::Libdrm)
-
-if(PIPEWIRE_VERSION VERSION_GREATER_EQUAL 0.3.60)
-  obs_status(STATUS "PipeWire 0.3.60+ found, enabling camera support")
-
-  target_sources(linux-pipewire PRIVATE camera-portal.c camera-portal.h)
-endif()
-
-set_target_properties(linux-pipewire PROPERTIES FOLDER "plugins")
+target_include_directories(linux-pipewire PRIVATE ${libdrm_include_directories})
+target_link_libraries(linux-pipewire PRIVATE OBS::libobs OBS::glad PipeWire::PipeWire gio::gio)
```

Comment:
Add a newline before? You typically space these out even if they're single lines.